packages/mattermost-redux/src/selectors/entities/apps.ts

```diff
@@ -36,3 +36,37 @@ export const makeAppBindingsSelector = (location: string) => {
         },
     );
 };
+
+export const makeRHSAppBindingSelector = (location: string) => {
+    return createSelector(
+        'makeRHSAppBindingSelector',
+        (state: GlobalState) => state.entities.apps.rhs.bindings,
+        (state: GlobalState) => appsEnabled(state),
+        (bindings: AppBinding[], areAppsEnabled: boolean) => {
+            if (!areAppsEnabled || !bindings) {
+                return [];
+            }
+
+            const headerBindings = bindings.filter((b) => b.location === location);
+            return headerBindings.reduce((accum: AppBinding[], current: AppBinding) => accum.concat(current.bindings || []), []);
+        },
+    );
+};
+
+export const getAppCommandForm = createSelector(
```

Comment:
Strictly speaking, since these two don't need to do any computation and just return raw store data, they don't need to use `createSelector` for memoization. And since they take an argument, you actually don't want to use it since having two places using it with different arguments would cause the memoization to fail since it only stores the last value returned.

Both of those can be solved by just making it into a regular function that doesn't use `createSelector` though